test/test_debugger.cpp

```diff
@@ -965,8 +968,14 @@ bool debugger_client::test_copy_to_globals()
     return global_var["value"] == local_var["value"] && global_var["type"] == local_var["type"];
 }
 
+void debugger_client::start()
+{
+    m_client.start();
+}
+
 void debugger_client::shutdown()
 {
+    m_client.stop_channels();
```

Comment:
Calling stop_channels here so that we could stop the process of polling the iopub and adding messages to the queue.